default/stringtables/en.txt

```diff
@@ -1186,6 +1186,9 @@ Start server in hostless mode. Server accepts players in the multiplayer lobby,
 OPTIONS_DB_SKIP_CHECKSUM
 Skip comparing the resource directory checksums.  This allows faster startup when client and server are on the same machine, using the same resource directory.
 
+OPTIONS_DB_TESTING
+Start server in testing mode. AI playing will be output log to colnsole instead file (Linux only).
```

Comment:
colnsole

Reply:
Option is no-op in other than Linux OS.

Reply:
Put the "Linux only" at the start.

And perhaps don't even add it as an option in those cases...

Reply:
Moved the option to macro condition for `FREEORION_LINUX`.